inc/class-statify.php

```diff
@@ -89,6 +89,9 @@ public function __construct() {
 			add_action( 'template_redirect', array( 'Statify_Frontend', 'track_visit' ) );
 			add_filter( 'query_vars', array( 'Statify_Frontend', 'query_vars' ) );
 			add_action( 'wp_footer', array( 'Statify_Frontend', 'wp_footer' ) );
+			if ( function_exists( 'is_amp_endpoint' ) ) { // Automattic AMP plugin present.
```

Comment:
I would see this statement, logic in a separate method and not logic inside the constructor. For me is more clear and on focus to switch to modern development we need to go to an empty constructor in context of usage from unit tests. You agree @tthemann and would you enhance them? So only fire this method currently via the constructor and the logic in a enhanced method.

Reply:
The constructor already does hack of a lot of stuff a constructor should not do...

I think we should eliminate the pointless `_constructor()` / `static instance()` combination entirely in favor of a single static `init()` method. _But not in the context of this PR._

Reply:
To be honest, these lines have been almost the only PHP code I wrote since the early 2000s - therefore I might not be the right person to discuss coding standards or philosophy ;-).

Reply:
PR #140 solve this so we should merge this after update for the new code base after merge #140.